lib/buildEmbed.js

```diff
@@ -10,6 +10,20 @@ module.exports = function(id, options, index) {
   return output;
 }
 
+function urlParams(options){
+  let params = []
+
+  // autoplay is _technically_ possible, but be cool, don't do this
+  if(options.allowAutoplay) params.push('autoplay=1');
+  if(options.recommendChannel) params.push('rel=0');
```

Comment:
I'd like to tweak the name of this option handle to make it more obviously a boolean choice...

```suggestion
  if(options.recommendSelfOnly) params.push('rel=0');
```